cfme/tests/automate/test_automate_manual.py

```diff
@@ -970,3 +927,62 @@ def test_user_requester_for_lifecycle_provision():
          1671563
     """
     pass
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+def test_remove_openshift_deployment_in_automate():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.11
+        casecomponent: Automate
+        tags: automate
+
+    Bugzilla:
+        1672937
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_vm_naming_number_padding():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: Automate
+        tags: automate
+
+    Bugzilla:
+        1688672
+    """
+    pass
```

Comment:
With my comment above, I see this is not for 5.11 but for 5.10. If you need this in, create a new PR just with this and the removal of the `test_custom_button_enable` and `test_custom_button_visibility` and any other changes.